test/main.test.js

```diff
@@ -97,51 +97,60 @@ describe('GotScraping', () => {
             expect(response.statusCode).toBe(200);
             expect(response.httpVersion).toBe('2.0');
         });
-        test('should work with proxyUrl and http2', async () => {
+        test('should work with proxyUrl and http1', async () => {
             jest.setTimeout(60000);
```

Comment:
Why such huge timeouts in the various tests? It's a http request, it should take a second at max.

Reply:
Missed this one after chaning the proxy groups.